uiowa/uiowa
# [Bug] Fix revision comparisons

Resolves #8231 




# How to test

- Sync a site, go to the revisions tab of a piece of content, and compare two revisions.
- See that it doesn't WSOD.
- Success!
- Try with different content types.
- You can also visit `/admin/config/content/diff/fields` and see that there are no errors. This page should error if there is any field on the site that does not have the necessary diff update

Try syncing and trying the above any sites that may have unique fields from different modules
